recipe/meta.yaml

```diff
@@ -12,14 +12,16 @@ source:
     - conda-build.patch
 
 build:
-  number: 0
+  number: 1
   skip: true  # [not win]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - {{ compiler('c') }}
     - {{ compiler('m2w64_fortran') }}
+  run:
+    - mpi 1.0 msmpi
```

Comment:
@mccarthyryanc On Conda-Forge, the meta package `mpi` serves as a mutex (see the PR title) so that it is guaranteed that there's only one MPI implementation installed per conda environment at a time. Without this, in theory (as per @isuruf, see the linked comment in the PR description) there could be multiple MPI installed on Windows. Presumably this is the reason that @isuruf wants to mark the old `msmpi` package as broken after this PR is in, since it didn't correctly acquire the mutex. (I am one of the `openmpi` maintainers, which is why I know this 🙂)

I just made sure the `msmpi` variant is available upstream: https://github.com/conda-forge/mpi-feedstock/pull/7, but perhaps it takes some time for the package index to propagate. I will kick off the CI shortly.

Reply:
ah, got it. I'm late to the party on this, struggling to catch up :rofl:

Reply:
If you're interested, this is where the mutex is done in other MPI packages:
- `openmpi`: https://github.com/conda-forge/openmpi-feedstock/blob/fab8f44a6212cbe1f1c6d896c4cc6a07eb20b2a7/recipe/meta.yaml#L41
- `mpich`: https://github.com/conda-forge/mpich-feedstock/blob/ee123f39d825a744f3edd69ed3216610888662bc/recipe/meta.yaml#L35

Reply:
Thanks! That does help!